hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/CacheAwareLoadBalancer.java

```diff
@@ -53,6 +55,12 @@
 public class CacheAwareLoadBalancer extends StochasticLoadBalancer {
   private static final Logger LOG = LoggerFactory.getLogger(CacheAwareLoadBalancer.class);
 
+  public static final String CACHE_RATIO_THRESHOLD = "hbase.master.balancer.stochastic.cacheRatio";
+
+  public static final String MOVE_THROTTLING = "hbase.master.balancer.move.throttlingMs";
```

Comment:
If throttle support is only added to the cache aware balancer, then maybe this configuration name should also be specific to the cache aware balancer

Reply:
Good point. Right now, it's only implemented by the cache aware balancer. But since it may be implemented by other possible balancer implementations in the future, I thought may be worth move these two to the LoadBalancer interface.